examples/tictactoe/src/components/loading-spinner/loading-spinner.tsx

```diff
@@ -0,0 +1,21 @@
+import { type FC } from "react";
+import "./styles.css";
+
+type LoadingSpinnerProps = {
+  isSmall?: boolean;
+};
+
+export const LoadingSpinner: FC<LoadingSpinnerProps> = ({ isSmall }) => {
+  return (
+    <div className={isSmall ? "lds-roller-small" : "lds-roller"}>
+      <div></div>
```

Comment:
Interesting way to implement the spinner, looks a bit hacky. I wonder if that's more efficient than an animated SVG like: https://github.com/n3r4zzurr0/svg-spinners/blob/main/svg-css/6-dots-rotate.svg